src/main/java/com/coveros/selenified/element/check/azzert/AssertExcludes.java

```diff
@@ -0,0 +1,186 @@
+     * class. If the element isn't present, this will constitute a failure, same
+     * as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param unexpectedClass - the unexpected class value
+     */
+    public void clazz(String unexpectedClass) {
+        String clazz = checkClazz(unexpectedClass, 0, 0);
+        assertNotNull(NO_ELEMENT_FOUND, clazz);
+        assertFalse("Class Mismatch: class of '" + clazz + CONTAINS + unexpectedClass + "'", clazz.contains(unexpectedClass));
+    }
+
+    /**
+     * Asserts that the element does not contain the provided expected
+     * attribute. If the element isn't present, this will constitute a failure,
+     * same as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param expectedAttribute - the attribute to check for
+     */
+    public void attribute(String expectedAttribute) {
+        String[] attributes = checkAttribute(expectedAttribute, 0, 0);
+        assertNotNull(NO_ELEMENT_FOUND, attributes);
+        assertFalse("Attribute found: element attributes of '" + String.join(",", attributes) +
+                CONTAINS + expectedAttribute + "'", Arrays.asList(attributes).contains(expectedAttribute));
+    }
+
+    /**
+     * Asserts that the element's text does not contain the provided expected
+     * text. If the element isn't present, this will constitute a failure, same
+     * as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param expectedText the expected text of the element
+     */
+    public void text(String expectedText) {
+        String text = checkText(expectedText, 0, 0);
+        assertNotNull(NO_ELEMENT_FOUND, text);
+        assertFalse("Text found: element text of '" + text + CONTAINS + expectedText + "'", text.contains(expectedText));
+    }
+
+    /**
+     * Asserts that the element's value does not contain the provided expected
+     * value. If the element isn't present or an input, this will constitute a
+     * failure, same as a mismatch. This information will be logged and
+     * recorded, with a screenshot for traceability and added debugging support.
+     *
+     * @param expectedValue the expected value of the element
+     */
+    public void value(String expectedValue) {
+        String value = checkValue(expectedValue, 0, 0);
+        if (value == null) {
+            String reason = NO_ELEMENT_FOUND;
+            if (getElement().is().present()) {
+                reason = "Element not input";
+            }
+            fail(reason);
+        }
+        assertNotNull(value);
+        assertFalse("Value found: element value of '" + value + CONTAINS + expectedValue + "'", value.contains(expectedValue));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "value" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS2259)